realm/realm-library/src/androidTest/java/io/realm/SortTest.java

```diff
@@ -153,19 +153,19 @@ private void checkSortTwoFieldsStringAscendingIntAscending(RealmResults<AllTypes
 
         assertEquals("Adam", results.get(0).getColumnString());
         assertEquals(4, results.get(0).getColumnLong());
-        assertEquals(2, ((TableView) results.getTable()).getSourceRowIndex(0));
+        assertEquals(2, ((TableView) results.getTableOrView()).getSourceRowIndex(0));
```

Comment:
That is even better, but since we don't do it as part of this PR 😄 . This was a rename based on the fact that `getTable()` was a very wrong name especially when hunting bugs where the distinction between an actual table and tableview was important.